pkg/services/ngalert/models/alert_rule.go

```diff
@@ -174,6 +174,10 @@ type AlertRuleGroupKey struct {
 	RuleGroup    string
 }
 
+func (k AlertRuleGroupKey) String() string {
+	return fmt.Sprintf("{orgID: %d, namespaceUID: %s, groupName: %s}", k.OrgID, k.NamespaceUID, k.RuleGroup)
```

Comment:
You can even omit the braces :)

Reply:
Agree. However, I just followed the same format as AlertRuleKey (right below)